src/rebar3_rust_compile_prv.erl

```diff
@@ -104,12 +104,11 @@ do_target(#{kind := Kind, name := Name}, CrateDir, OutDir, State) ->
                         true -> {" --release", "release"};
                         false -> {"", "debug"}
                     end,
-    VerboseSwitch = " --verbose",
```

Comment:
Can we implement this just as `VerboseSwitch = ""`?  Keeping this variable will serve as a reminder to add controllable verboseness at some point in the future.